packages/react-aria-components/docs/DisclosureGroup.mdx

```diff
@@ -81,6 +86,128 @@ import {UNSTABLE_DisclosureGroup as DisclosureGroup, UNSTABLE_Disclosure as Disc
 @import './Disclosure.mdx' layer(disclosure);
 ```
 
+## Features
+
+
+Custom styled disclosure groups can be difficult to implement in an accessible way with the correct aria pattern. When implementing, you must manually ensure that the button and panel are semantically connected via ids for accessibility. `DisclosureGroup` helps with this and other accessibility features while allowing for custom styling.
+
+* **Accessible** - Disclosure group is exposed to assistive technology via ARIA 
+* **Styleable** - Disclosures include builtin states for styling such as keyboard focus, disabled, and expanded states.
+
+## Anatomy
+
+<Anatomy />
+
+A disclosure group consists of a set of disclosures. Each disclosure includes a button within a heading and panel of content that is either shown or hidden. Zero or more disclosures within a group can be expanded at the same time, however, by default, only one disclosure can be expanded at a time. Users may click or touch a disclosure to expand it, or use the <Keyboard>Tab</Keyboard> key to navigate between disclosures and the <Keyboard>Enter</Keyboard> or <Keyboard>Space</Keyboard> key to toggle it.
+
+```tsx
+import {UNSTABLE_DisclosureGroup as DisclosureGroup, UNSTABLE_Disclosure as Disclosure, Button, UNSTABLE_DisclosurePanel as DisclosurePanel} from 'react-aria-components';
+
+<DisclosureGroup>
+  <Disclosure>
+    <h3>
+      <Button />
+    </h3>
+    <DisclosurePanel />
+  </Disclosure>
+</DisclosureGroup>
+```
+
+### Composed Components
+
+<section className={styles.cardGroup} data-size="small">
+
+<ExampleCard
```

Comment:
looks like it may be overflowing the container

Reply:
it looks okay to me? mind sharing a screenshot

Reply:
okay actually i just added some inline styling to those svg's so maybe that'll help?